libsolidity/analysis/NameAndTypeResolver.cpp

```diff
@@ -450,7 +450,7 @@ vector<T const*> NameAndTypeResolver::cThreeMerge(list<list<T const*>>& _toMerge
 	{
 		for (auto it = _toMerge.begin(); it != _toMerge.end();)
 		{
-			it->remove(_candidate);
+			it->restd::move(_candidate);
```

Comment:
Looks like your search&replace regex was a bit too weak :)
```suggestion
			it->remove(_candidate);
```